### DISCUSSION TITLE
Help adding support for rnix_lsp

### DISCUSSION THREAD

**Discussion by ZeMoreo** [ray-x/navigator.lua#113] [Q&A]
I was trying to add support for rnix_lsp in my branch of navigator.lua, but for some reason when I tried to test it, rnix_lsp won't start and work. 

Fork: https://github.com/Loreo13/navigator.lua

**Comment by ray-x** [comment]
I think one issue is rnix_lsp is not supported by lspconfig.

In clients.lua

```
  log(ft, client, loaded)
  if lspconfig[client] == nil then
    log("not supported by nvim", client)
    return
  end
```

You may enable log by setting:`debug=true`

**Comment by ZeMoreo** [comment]
It seems to be supported by lspconfig. https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/server_configurations/rnix.lua

**Reply by ray-x** [reply]
try `rnix` instead of `rnix_lsp` as the lsp name?
